.github/workflows/build-using-toolchain-bundle.yml

```diff
@@ -0,0 +1,59 @@
+name: Build and test app in toolchain bundle environment
+
+on:
+  pull_request:
+
+jobs:
+  build-and-test-in-toolchain-bundle:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout repository with example application
+        uses: actions/checkout@v4
+        with:
+          path: example-application
+
+      - name: Prepare west project
+        run: |
+          python3 -m pip install west
+          west init -l example-application
+          west update -o=--depth=1 -n
+
+      - name: Install nrfutil and toolchain manager
```

Comment:
Would it make sense to take lines 21..49 and introduce that as a new action? So in here it would be: nrfconnect/action-install-toolchain@main